setup.py

```diff
@@ -72,6 +72,6 @@
         packages=["qcore", "qcore.tests"],
         package_data={"qcore": DATA_FILES},
         ext_modules=EXTENSIONS,
-        setup_requires=["Cython"],
-        install_requires=["Cython"],
+        setup_requires=["Cython==0.29.36"],
+        install_requires=["Cython==0.29.36"],
```

Comment:
We probably don't need to pin here, just when building. This would make it so that no one can import qcore in an env that has a different version of Cython, which is too restrictive.

Reply:
```suggestion
        install_requires=["Cython"],
```
Let's try this